src/menu/ImGuiImpl.cpp

```diff
@@ -383,7 +387,8 @@ bool SupportsViewports() {
     switch (impl.backend) {
         case Backend::DX11:
             return true;
-        case Backend::SDL:
+        case Backend::SDL_OPENGL:
+        case Backend::SDL_METAL:
```

Comment:
it seems we have some inconsistencies around when we ifdef and when we don't, it seems like we should ifdef away DX11 when not on windows but that's not happening here, so maybe not ifdefing away metal when not on mac is fine too?